wrapper/src/main/java/software/amazon/jdbc/plugin/readwritesplitting/ReadWriteSplittingPlugin.java

```diff
@@ -326,15 +326,18 @@ void switchConnectionIfRequired(final boolean readOnly) throws SQLException {
           switchToReaderConnection(hosts);
         } catch (final SQLException e) {
           if (!isConnectionUsable(currentConnection)) {
-            logAndThrowException(Messages.get("ReadWriteSplittingPlugin.errorSwitchingToReader"),
-                SqlState.CONNECTION_UNABLE_TO_CONNECT, e);
+            logAndThrowException(
+                Messages.get("ReadWriteSplittingPlugin.errorSwitchingToReader", new Object[] { e.getMessage() }),
+                SqlState.CONNECTION_UNABLE_TO_CONNECT,
+                e);
             return;
           }
 
-          // Failed to switch to a reader; use writer as a fallback
-          LOGGER.warning(() -> Messages.get(
+          // Failed to switch to a reader. {0}. The current writer will be used as a fallback: ''{1}''
```

Comment:
this comment uses {0} and {1} which will not print anything
```suggestion
          // Failed to switch to a reader. The current writer will be used as a fallback.
```

Reply:
There're two parameters are passed to `Messages.get()`: error message and connection host name.